src/client/rest-client.js

```diff
@@ -42,12 +42,16 @@ class RestClient extends Client {
       });
   }
 
-  callFunction (name, data) {
+  callFunction (name, data, resource, auth) {
```

Comment:
Instead of adding two new positional arguments, let's add a single `opts` positional argument on which you can attach any "keyword" arguments you need, such as `resource` and `auth`. This will make this method more future-proof.